src/graph/session/ClientSession.h

```diff
@@ -23,8 +23,15 @@ struct SpaceInfo {
   meta::cpp2::SpaceDesc spaceDesc;
 };
 
+// A ClientSession is equivalent to a connection between a client and a server.
```

Comment:
Session is not equivalent to connection. E.G. session could be saved in cluster even after connection broken.